src/modules/localeq/localeq.qml

```diff
@@ -119,9 +79,18 @@ Page
                     Layout.alignment: Qt.AlignRight|Qt.AlignVCenter
                     Layout.columnSpan: 2 
                     text: qsTr("Change")
-                    onClicked: console.log("Adjust Locale clicked");
+                    onClicked: { 
+                        onClicked: load.source = "i18n.qml"
+                    }
                 }
             }
         }
+        
+        
+        
     }
+    Loader { 
```

Comment:
Current state of localeq with this commit:
![Screenshot_20200418_171001](https://user-images.githubusercontent.com/5360485/79641422-c5fda280-8197-11ea-8895-aa952c6dea79.png)
![Screenshot_20200418_171021](https://user-images.githubusercontent.com/5360485/79641426-ca29c000-8197-11ea-8b55-c259d2d591c9.png)